.github/workflows/daily_common.yml

```diff
@@ -86,4 +91,7 @@ jobs:
         run: |
           nim --version
           nimble --version
-          NIMFLAGS="${NIMFLAGS} --mm:${{ matrix.nim.memory_management }}" nimble test
+          [[ ${{ inputs.use_sat_solver }} ]] && dependency_solver="sat" || dependency_solver="legacy"
```

Comment:
```suggestion
    if [[ "${{ inputs.use_sat_solver }}" == "true" ]]; then
      dependency_solver="sat"
    else
      dependency_solver="legacy"
    fi
```

Reply:
Any reason you oppose to ternary?

Reply:
It's more readable in this way.

Reply:
But just a suggestion, up to you

Reply:
Actually, you're right; that ternary is a bit convoluted.